package/upgrade/upgrade_manifests.sh

```diff
@@ -453,27 +531,16 @@ upgrade_logging_event_audit() {
     # reuse frame work to generate yaml file
     upgrade_managed_chart_from_version $UPGRADE_PREVIOUS_VERSION rancher-logging rancher-logging.yaml
 
-    echo "Apply resource file"
+    echo "Apply resource file of logging and audit"
 
     kubectl apply -f ./rancher-logging.yaml
 
     # wait for managedchart to be applied
     sleep 50
```

Comment:
just curious, If we have the retry mechanism on the following. Do we still need to sleep 50 seconds here?

Reply:
The consideration:
(1) The first sleep will help reduce looping message. From practice, the `managedchart` may take 30 seconds ~3 minutes to be applied. The loop waiting interval is 10s. 
(2) The fix tries ot change exising code as few as possible.

Reply:
OK, thanks for the explanation!